tools/collect_assets_type.py

```diff
@@ -0,0 +1,34 @@
+import json
+from argparse import ArgumentParser
+
+from pydantic.json import pydantic_encoder
+
+from src.helper import collect_assets_typeid
+
+
+def write_result_to_file(asset_types: dict):
+    with open("metadata.json", "w") as f:
+        json.dump(asset_types, f, indent=4, default=pydantic_encoder)
+
+
+if __name__ == "__main__":
+    parser = ArgumentParser()
+    parser.add_argument("-c", "--collibraInstance")
+    parser.add_argument("-u", "--username")
+    parser.add_argument("-p", "--password")
+    parser.add_argument("-a", "--applicationName")
```

Comment:
might be good to provide the 'help' argument here, especially for the not obvious ones like applicationName and assetType:

```
parser.add_argument(
        "target_directory", help="Target directory in which the files for batch custom lineage will be stored"
    )
```